cypress/e2e/content/article-links.cy.js

```diff
@@ -64,7 +64,10 @@ describe('Article links', () => {
       // Test internal links (including anchor links)
       cy.get('article a[href^="/"]').each(($a) => {
         const href = $a.attr('href');
-        testLink(href);
+        // Skip links that contain "kapa.ai"
```

Comment:
Thanks for noting this! I'll keep this until I come up with something better--we only want to check content links for now, so this is fine. I just recently added Cypress and am working out the kinks.